web/app/styles/help.css

```diff
@@ -150,6 +150,6 @@ code {
     content: '';
     display: block;
     height: 100px;
-    background: url(images/logo.png) center / contain no-repeat #004165 !important; /* Brand color. */
+    background: url(/images/logo.png) center / contain no-repeat #004165 !important; /* Brand color. */
```

Comment:
An absolute URL! This would have problems on demo.lynxkite.com. Let's check what's the right way to handle that.

Reply:
> An absolute URL! This would have problems on demo.lynxkite.com. Let's check what's the right way to handle that.

Setting Vite's base URL to the empty string makes everything relative! This URL, for example, becomes `url(../images/logo.png)` which seems correct.